tests/test_basic.py

```diff
@@ -1711,6 +1711,8 @@ def test_getFeatureNames():
     test_data_path = testdata_dir.parent / 'featurenames.json'
     with open(test_data_path, 'r') as featurenames_json:
         expected_featurenames = json.load(featurenames_json)
+    # add the new names for the deprecated ones
+    expected_featurenames += ["spike_count", "spike_count_stimint"]
```

Comment:
The featurenames_json should reflect all the features that we have. If we don't want to duplicate spike_count, it would be better I think to have spike_count in the file, and add deprecated Spikecount in code here to legacy, instead of the inverse as we have now

Reply:
Ok makes sense so renaming Spikecount -> spike_count in the file.

Reply:
Done in the last push.